BPBackendDjango/BPBackendDjango/Views/userviews.py

```diff
@@ -1345,6 +1401,15 @@ def post(self, request, *args, **kwargs):
                     'data': {}
                 }
             return Response(data)
+
+        #check length
+        if (len(req_data['street']) > STREET_LENGTH) or (len(req_data['postal_code']) > POSTAL_CODE_LENGTH) or (len(req_data['country']) > COUNTRY_LENGTH) or (len(req_data['city']) > CITY_LENGTH) or (len(req_data['house_nr']) > H_NR_LENGTH) or (len(req_data['address_add']) > ADDRESS_ADD_LENGTH):
```

Comment:
Split this check into separate checks to provide better feedback. Maybe just create a method to check an argument for a given length and another one to create a response with a fitting description.

Reply:
Then you can reuse these methods above.

Reply:
& on other occasions where lengths are checked